### DISCUSSION TITLE
Detection Engine

### DISCUSSION THREAD

**Discussion by peasead** [Security-Onion-Solutions/securityonion#4841] [Unsupported Versions] [answered]
Has anyone gotten the Detection Engine to work in SO?

Enabling the engine is trivial, but has anyone changed the indices or index patterns that the Detection Engine uses so that the data from `*:so-*` can be used?

**Comment by weslambert** [comment, marked as answer]
Hi @peasead ,

You should be able to change the value(s) for `securitySolution:defaultIndex` in Kibana's Advanced Settings, however, it may get overwritten at some point after changing.

**Reply by peasead** [reply]
Yep, there it is.

`Stack Management -> Kibana -> Advanced Settings -> Security Solution`

![image](https://user-images.githubusercontent.com/7442091/126405532-bca61715-5464-4507-9c45-0f7b853a98c6.png)
